packages/cactus-test-tooling/src/test/typescript/integration/quorum/quorum-test-ledger/constructor-validates-options.test.ts

```diff
@@ -7,48 +7,56 @@ import {
   isIKeyPair,
 } from "../../../../../main/typescript/public-api";
 
-tap.test("constructor throws if invalid input is provided", (assert: any) => {
-  assert.ok(QuorumTestLedger);
-  assert.throws(() => new QuorumTestLedger({ containerImageVersion: "nope" }));
-  assert.end();
-});
+tap.test(
+  "constructor throws if invalid input is provided",
+  (assert: unknown) => {
```

Comment:
@yayoiukai2021 I recommend here to change the assert parameter's type to `Test` instead of `unknown`.
(and recommending the same in all other instances where you changed the type of the same parameter in other test cases)